tools/donate-cpu-server.py

```diff
@@ -422,7 +422,7 @@ def diffReport(resultsPath: str) -> str:
         for messageId in data['sums']:
             sums = data['sums'][messageId]
             if OLD_VERSION not in sums:
-                continue
+                break
```

Comment:
We can just bail out on the first occurrence of an old version because if one entry is outdated all of them are.